packages/grafana-ui/src/components/Table/TableNG/utils.ts

```diff
@@ -128,42 +128,78 @@ export function getMaxWrapCell(
  * Returns true if text overflow handling should be applied to the cell.
  */
 export function shouldTextOverflow(field: Field): boolean {
-  let type = getCellOptions(field).type;
-
-  return (
-    field.type === FieldType.string &&
+  const cellOptions = getCellOptions(field);
```

Comment:
nit: could leave this as it was, since we only read `type` from here. it makes the code below shorter.